### ISSUE TITLE
Create resources guide for playbook

### ISSUE THREAD

**Issue by selynna** [hackedu/hack-camp#410]
Mainly git blame for now?

**Comment by zachlatta** [comment]
What do you mean by this?

On Sat, Aug 8, 2015 at 3:33 PM, Selynna Sun notifications@github.com
wrote:

> Mainly git blame for now?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/issues/410.

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by maxwofford** [comment]
`19:20:00 <ssun> er people caught typos so i taught some people git blame and i figured it would be a good idea to write up something people can refer to`

**Comment by zachlatta** [comment]
Mmm, gotcha. How is `git blame` useful in the case of fixing the typos, out
of curiosity?

On Sun, Aug 9, 2015 at 1:41 PM, Max Wofford notifications@github.com
wrote:

> 19:20:00 <ssun> er people caught typos so i taught some people git blame
> and i figured it would be a good idea to write up something people can
> refer to
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/issues/410#issuecomment-129236323.

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by selynna** [comment]
@zachlatta they can leave line notes so we can see where we have typos, and fix them accordingly. Does that make sense at all...?

**Comment by zachlatta** [comment]
I think I'm still a bit confused.

`git blame` shows you which commit added each line of code. Here's an excerpt from `ACTIVITIES.md`:

``````
96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03 01:16:10 -0700  123) # Energizers
96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03 01:16:10 -0700  124) 
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  125) ## Bad startup ideas
96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03 01:16:10 -0700  126) 
96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03 01:16:10 -0700  127) ```
96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03 01:16:10 -0700  128) Time: 10 min
fc6489e2 cohort_4/playbook/ACTIVITIES.md (Zach Latta     2015-08-03 10:03:42 -0700  129) Materials: none
fc6489e2 cohort_4/playbook/ACTIVITIES.md (Zach Latta     2015-08-03 10:03:42 -0700  130) Team sizes:
96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03 01:16:10 -0700  131) 
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  132) - when 2-8 people, group into 2 teams
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  133) - when 8+, group into teams no larger than 4
96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03 01:16:10 -0700  134) ```
96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03 01:16:10 -0700  135) 
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  136) - For the purposes of this example I will refer to `team A` and `team B`.
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  137) - Each team will break up and be given 2 minutes to come up with a crazy startup
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  138) idea.
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  139) - After the teams have ideas team A will take the role of investor and team B
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  140) will be a startup.
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  141) - Team A tells team b what startup idea they will be pitching.
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  142) - Members of team B come up with and announce their positions and roles at the
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  143)   company they are representing.
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  144) - For 3 minutes team B pitches the company.
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  145) 
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  146) > team A: You are pitching tinder for dogs
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  147) > someone on team B: I am the Chief Executive Director intern
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  148) > someone else on team B: I am a hot single dog and beta tester
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  149) > team B: We're here to tell you about the next big thing... tinder for dogs
6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford    2015-08-06 22:16:06 -0700  150) > ~pitching continues for 3 minutes
``````

How does this let them leave line notes?

**Comment by selynna** [comment]
When you click on the specific commit you can scroll to the line you need
changed and add line notes, right?

**Comment by zachlatta** [comment]
Ah – that's just browsing a commits's diff/tree on GitHub (which isn't a
feature of git). git blame is actually part of git.

On Sun, Aug 9, 2015 at 2:43 PM, Selynna Sun notifications@github.com
wrote:

> When you click on the specific commit you can scroll to the line you need
> changed and add line notes, right?
> 
> On Sun, Aug 9, 2015 at 2:39 PM Zach Latta notifications@github.com
> wrote:
> 
> > I think I'm still a bit confused.
> > 
> > git blame shows you which commit added each line of code. Here's an
> > excerpt from ACTIVITIES.md:
> > 
> > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > 01:16:10 -0700 123) # Energizers
> > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > 01:16:10 -0700 124)
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 125) ## Bad startup ideas
> > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > 01:16:10 -0700 126)
> > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > 01:16:10 -0700 127) `
> > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > 01:16:10 -0700 128) Time: 10 min
> > fc6489e2 cohort_4/playbook/ACTIVITIES.md (Zach Latta 2015-08-03 10:03:42
> > -0700 129) Materials: none
> > fc6489e2 cohort_4/playbook/ACTIVITIES.md (Zach Latta 2015-08-03 10:03:42
> > -0700 130) Team sizes:
> > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > 01:16:10 -0700 131)
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 132) - when 2-8 people, group into 2 teams
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 133) - when 8+, group into teams no larger than 4
> > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > 01:16:10 -0700 134)`
> > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > 01:16:10 -0700 135)
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 136) - For the purposes of this example I will refer to
> > `team A` and `team B`.
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 137) - Each team will break up and be given 2 minutes to
> > come up with a crazy startup
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 138) idea.
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 139) - After the teams have ideas team A will take the role
> > of investor and team B
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 140) will be a startup.
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 141) - Team A tells team b what startup idea they will be
> > pitching.
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 142) - Members of team B come up with and announce their
> > positions and roles at the
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 143) company they are representing.
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 144) - For 3 minutes team B pitches the company.
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 145)
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 146) > team A: You are pitching tinder for dogs
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 147) > someone on team B: I am the Chief Executive Director
> > intern
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 148) > someone else on team B: I am a hot single dog and
> > beta tester
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 149) > team B: We're here to tell you about the next big
> > thing... tinder for dogs
> > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > 22:16:06 -0700 150) > ~pitching continues for 3 minutes
> > 
> > How does this let them leave line notes?
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/hackedu/hack-camp/issues/410#issuecomment-129243913
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/issues/410#issuecomment-129244109.

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by selynna** [comment]
Oh okay that makes sense.  But you can use git blame to get to the commit
tree and add line notes, right?

**Comment by zachlatta** [comment]
All git blame does is annotate the file. See
http://git-scm.com/docs/git-blame for the specifics.

On Sun, Aug 9, 2015 at 3:05 PM, Selynna Sun notifications@github.com
wrote:

> Oh okay that makes sense. But you can use git blame to get to the commit
> tree and add line notes, right?
> 
> On Sun, Aug 9, 2015 at 3:00 PM Zach Latta notifications@github.com
> wrote:
> 
> > Ah – that's just browsing a commits's diff/tree on GitHub (which isn't a
> > feature of git). git blame is actually part of git.
> > 
> > On Sun, Aug 9, 2015 at 2:43 PM, Selynna Sun notifications@github.com
> > wrote:
> > 
> > > When you click on the specific commit you can scroll to the line you
> > > need
> > > changed and add line notes, right?
> > > 
> > > On Sun, Aug 9, 2015 at 2:39 PM Zach Latta notifications@github.com
> > > wrote:
> > > 
> > > > I think I'm still a bit confused.
> > > > 
> > > > git blame shows you which commit added each line of code. Here's an
> > > > excerpt from ACTIVITIES.md:
> > > > 
> > > > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > > > 01:16:10 -0700 123) # Energizers
> > > > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > > > 01:16:10 -0700 124)
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 125) ## Bad startup ideas
> > > > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > > > 01:16:10 -0700 126)
> > > > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > > > 01:16:10 -0700 127) `
> > > > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > > > 01:16:10 -0700 128) Time: 10 min
> > > > fc6489e2 cohort_4/playbook/ACTIVITIES.md (Zach Latta 2015-08-03
> > > > 10:03:42
> > > > -0700 129) Materials: none
> > > > fc6489e2 cohort_4/playbook/ACTIVITIES.md (Zach Latta 2015-08-03
> > > > 10:03:42
> > > > -0700 130) Team sizes:
> > > > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > > > 01:16:10 -0700 131)
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 132) - when 2-8 people, group into 2 teams
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 133) - when 8+, group into teams no larger than 4
> > > > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > > > 01:16:10 -0700 134)`
> > > > 96618db9 cohort_4/playbook/ACTIVITIES.md (Jonathan Leung 2015-08-03
> > > > 01:16:10 -0700 135)
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 136) - For the purposes of this example I will refer to
> > > > `team A` and `team B`.
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 137) - Each team will break up and be given 2 minutes to
> > > > come up with a crazy startup
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 138) idea.
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 139) - After the teams have ideas team A will take the
> > > > role
> > > > of investor and team B
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 140) will be a startup.
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 141) - Team A tells team b what startup idea they will
> > > > be
> > > > pitching.
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 142) - Members of team B come up with and announce their
> > > > positions and roles at the
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 143) company they are representing.
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 144) - For 3 minutes team B pitches the company.
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 145)
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 146) > team A: You are pitching tinder for dogs
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 147) > someone on team B: I am the Chief Executive
> > > > Director
> > > > intern
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 148) > someone else on team B: I am a hot single dog and
> > > > beta tester
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 149) > team B: We're here to tell you about the next big
> > > > thing... tinder for dogs
> > > > 6695213b cohort_4/playbook/ACTIVITIES.md (Max Wofford 2015-08-06
> > > > 22:16:06 -0700 150) > ~pitching continues for 3 minutes
> > > > 
> > > > How does this let them leave line notes?
> > > > 
> > > > —
> > > > Reply to this email directly or view it on GitHub
> > > > <
> > > > https://github.com/hackedu/hack-camp/issues/410#issuecomment-129243913
> > > > .
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> > > https://github.com/hackedu/hack-camp/issues/410#issuecomment-129244109
> > > .
> > 
> > ## 
> > 
> > Zach Latta
> > hackEDU | zach@hackedu.us
> > 
> > —
> > 
> > Reply to this email directly or view it on GitHub
> > <https://github.com/hackedu/hack-camp/issues/410#issuecomment-129252999
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/issues/410#issuecomment-129254637.

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by jonleung** [comment]
@ssun098 and @zachlatta, what is the status of this?

I am closing this for now so that we can meet this objective: https://github.com/hackedu/hack-camp/issues/498.

Please feel free to reopen!

**Comment by selynna** [comment]
@jonleung I submitted a pr a long time ago

**Comment by jonleung** [comment]
@ssun098, ahh yes I remember! Our bad for failing to be on top of that! Did the pull request get merged? Also, can you please link the pull request? Thanks!

**Comment by selynna** [comment]
@jonleung just kidding it's a branch called "resources"

**Comment by jonleung** [comment]
@ssun098, cool, I didn't see a pull request for it yet. Just create a pull request if you want us to merge it in!

**Comment by selynna** [comment]
Created and assigned you to it

**Comment by jonleung** [comment]
Reopened issue because this issue is actively being worked on.

**Comment by maxwofford** [comment]
This is closed by #519

**Comment by jonleung** [comment]
:fireworks:
